### PR TITLE
Update openshift-knative/serving OCP versions

### PR DESCRIPTION
**Pull request by ReToCode**
JIRA: https://issues.redhat.com/browse/SRVCOM-2303

**Changes**
- Use OCP 4.12 for Serving 1.6, 1.7, 1.8 as upper bound
- Drop OCP 4.8 and OCP 4.9 builds for Serving 1.8

### CODE REVIEW

**Comment by ReToCode** [comment]
/pj-rehearse pull-ci-openshift-knative-serving-release-v1.7-412-e2e-aws-ocp-412

File: ci-operator/jobs/openshift-knative/serving/openshift-knative-serving-release-v1.7-periodics.yaml [hunk 1]
```diff
@@ -11,21 +11,10 @@ periodics:
     path_alias: knative.dev/serving
     repo: serving
   labels:
-    ci-operator.openshift.io/variant: "411"
+    ci-operator.openshift.io/variant: "412"
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-knative-serving-release-v1.7-411-e2e-aws-ocp-411-continuous
-  reporter_config:
```
Comment (mgencur) [inline ci-operator/jobs/openshift-knative/serving/openshift-knative-serving-release-v1.7-periodics.yaml:18]: The slack notifications are removed here. Is this intentional? Sometimes they have to be manually added back after running "make jobs"

File: ci-operator/jobs/openshift-knative/serving/openshift-knative-serving-release-v1.8-periodics.yaml [hunk 2]
```diff
@@ -86,21 +75,10 @@ periodics:
     path_alias: knative.dev/serving
     repo: serving
   labels:
-    ci-operator.openshift.io/variant: "411"
+    ci-operator.openshift.io/variant: "412"
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-knative-serving-release-v1.8-411-e2e-aws-ocp-tls-411
```
Comment (mgencur) [inline ci-operator/jobs/openshift-knative/serving/openshift-knative-serving-release-v1.8-periodics.yaml:92]: Same here, is removing slack notif intentional?

**Comment by mgencur** [comment]
/lgtm
Please unhold when ready.

**Pull request merged**